.github/auto-release.yml

```diff
@@ -29,7 +29,10 @@ categories:
       - 'bug'
       - 'hotfix'
 
-change-template: '- $TITLE @$AUTHOR (#$NUMBER)'
+change-template: |
+  ## $TITLE @$AUTHOR (#$NUMBER)
```

Comment:
So I considered doing this, but our descriptions have `##` in them usually. 

![image](https://user-images.githubusercontent.com/52489/90301000-03c2bd00-de52-11ea-8720-c09ea880e55b.png)

See #241

Reply:
https://github.com/cloudposse/build-harness/releases/tag/0.40.0